internal/signer/linux/pkcs11/pkcs11_test.go

```diff
@@ -155,12 +88,7 @@ func TestEncrypt(t *testing.T) {
 	defer key.Close()
 	msg := "Plain text to encrypt"
 	bMsg := []byte(msg)
-	// Softhsm only supports SHA1
-	key, err := key.WithHash(crypto.SHA1)
-	if err != nil {
-		t.Errorf("WithHash error: %q", err)
-	}
-	_, err = key.Encrypt(bMsg)
+	_, err := key.Encrypt(bMsg)
```

Comment:
What's the context for this?

Reply:
I changed this because the crypto encryptOAEP does not need SHA1 to encrypt successfully. As we are not trying to decrypt the cipher-text, I thought it was unnecessary to set SHA1 as the hash function.